src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -172,7 +172,7 @@ const TopicFragment = ({
     );
   };
   const renderDiscoveryItem = ({from, item, index}) => {
-    if (item.separator && route.name !== 'Followings') {
+    if (item.separator) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional check for `route.name !== 'Followings'` has been removed. This could potentially lead to a change in behavior if the `renderDiscoveryItem` function is expected to behave differently based on the route name. Please ensure that this change does not introduce unexpected behavior.

```diff
-    if (item.separator && route.name !== 'Followings') {
+    if (item.separator) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->